game/src/main/java/org/apollo/game/model/entity/Player.java

```diff
@@ -724,7 +735,8 @@ public void send(Message message) {
 	 * Sends the initial messages.
 	 */
 	public void sendInitialMessages(DisplayMode mode) {
-		send(new RebuildNormalMessage(position, index, world.getRegionRepository().getXteaRepository(), false));
+		send(new RebuildNormalMessage(position, index, updateInfo, world.getPlayerRepository(),
```

Comment:
This is an odd way to pass data for initialization, cc @Major-, we need to look at setting up player "pre-init" code.

Reply:
I wasn't sure how to pass it around. You could do everything with a single player reference but I tried to keep it consistent with how other things are done.